modules/influxdb/influxdb.go

```diff
@@ -118,6 +119,65 @@ func WithConfigFile(configFile string) testcontainers.CustomizeRequestOption {
 	}
 }
 
+// InfluxDBV2Config contains the configuration for InfluxDB v2.
+type InfluxDBV2Config struct {
```

Comment:
suggestion: let's not include the package name into the type to avoid redundancy in calls (i.e. `influxdb.ConfigV2` Vs `influxdb.InfluxDBV2Config`)
```suggestion
type ConfigV2 struct {
```